src/main/java/uk/gov/companieshouse/service/DissolutionService.java

```diff
@@ -25,4 +26,8 @@ public DissolutionCreateResponse create(DissolutionCreateRequest body, String co
     public boolean doesDissolutionRequestExistForCompany(String companyNumber) {
         return repository.findByCompanyNumber(companyNumber).isPresent();
     }
+
+    public DissolutionGetResponse get(String companyNumber) {
+        return creator.get(companyNumber);
```

Comment:
`creator.get`? Doesn't make sense. The creator's single responsibility is to create the dissolution. Nothing else. Either stick the implementation in here, or create a dedicated DissolutionReadService or something.